items.json

```diff
@@ -115234,78 +115234,55 @@
         "accuracy": 0,
         "missileSpeed": 0,
         "stackAmount": 0,
-        "length": 186,
+        "length": 179,
         "balance": 0.0,
-        "handling": 64,
+        "handling": 71,
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon",
           "WideGrip"
         ],
-        "thrustDamage": 27,
+        "thrustDamage": 26,
         "thrustDamageType": "Pierce",
-        "thrustSpeed": 82,
-        "swingDamage": 15,
-        "swingDamageType": "Blunt",
-        "swingSpeed": 24
+        "thrustSpeed": 89,
+        "swingDamage": 0,
+        "swingDamageType": "Undefined",
+        "swingSpeed": 39
       },
       {
         "class": "TwoHandedPolearm",
         "itemUsage": "polearm_block_shield_swing_thrust",
         "accuracy": 0,
         "missileSpeed": 0,
         "stackAmount": 0,
-        "length": 186,
-        "balance": 0.05,
-        "handling": 61,
-        "bodyArmor": 0,
-        "flags": [
-          "MeleeWeapon",
-          "NotUsableWithOneHand",
-          "WideGrip",
-          "TwoHandIdleOnMount"
-        ],
-        "thrustDamage": 27,
-        "thrustDamageType": "Pierce",
-        "thrustSpeed": 90,
-        "swingDamage": 15,
-        "swingDamageType": "Blunt",
-        "swingSpeed": 71
-      },
-      {
-        "class": "TwoHandedPolearm",
-        "itemUsage": "polearm_block_shield_thrust",
-        "accuracy": 0,
-        "missileSpeed": 0,
-        "stackAmount": 0,
-        "length": 186,
-        "balance": 0.05,
-        "handling": 61,
+        "length": 179,
+        "balance": 0.36,
+        "handling": 69,
         "bodyArmor": 0,
```

Comment:
How does that lead to more diversity , when you're just removing an extra optional mode?!

Reply:
also you removed the 2D mode not the 4d...
Currently this spear has in 2h mode : 4d and 2d. You just removed one mod? 
Are you like trying to nerf or make unfun specific classes on purpose?
If you continue going completely against the design philosophy of item balancing despite me telling you repeatedly to stop doing it , i'm going to start denying pull request on sight